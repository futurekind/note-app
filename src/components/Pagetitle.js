import React from 'react';
import styled from 'styled-components';

const View = styled.h1`
    margin: 0;
    font-size: 36px;
    letter-spacing: 1.5px;
    text-align: right;
    text-transform: uppercase;
    text-shadow: 0 1px 0 #FFFFFF;
`

const Pagetitle = ({
    children
}) => {
    return (
        <View>{ children }</View>
    )
}

export default Pagetitle