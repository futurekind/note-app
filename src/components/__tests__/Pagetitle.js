import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Pagetitle from '../Pagetitle';

describe('Pagetitle', () => {
    let wrapper;

    beforeEach(() => {
        wrapper = shallow(
            <Pagetitle />
        )
    })

    it('renders', () => {
        expect(toJSON(wrapper)).toMatchSnapshot();
    })

    it('renders children', () => {
        const wrapper = shallow(
            <Pagetitle>Some Title</Pagetitle>
        )
        expect(toJSON(wrapper)).toMatchSnapshot();
    })

})