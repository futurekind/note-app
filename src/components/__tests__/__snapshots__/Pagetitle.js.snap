exports[`Pagetitle renders 1`] = `<styled.h1 />`;

exports[`Pagetitle renders children 1`] = `
<styled.h1>
  Some Title
</styled.h1>
`;
